docs/api/agents.md

```diff
@@ -426,7 +426,7 @@ curl -X POST http://coder-server:8080/api/v2/workspaceagents/me/report-stats \
 
 ```json
 {
-  "report_interval": 0
+  "report_interval": -9223372036854776000
```

Comment:
For some reason the default value for integer is -Inf :)

Reply:
I believe we can fix it by adding the `example` annotation. Unfortunately, we need to add it to every `int` field?

Reply:
I don't think it's needed for every int, it only seems to apply to those that are `time.Duration`.

Reply:
You're right. Adding a record to `.swaggo` fixed the `time.Duration` issue. Thanks!